appveyor.yml

```diff
@@ -1,15 +0,0 @@
-build: off
-
-clone_folder: c:\gopath\src\github.com\gomods\athens
-
-environment:
-  GOPATH: c:\gopath
-  GO111MODULE: on
-  GOPROXY: https://proxy.golang.org
-  SKIP_UNTIL_113: true
-
-stack: go 1.13
-
-test_script:
-  - go test ./...
-
```

Comment:
@ClaytonNorthey92 we still need to run at least unit tests on windows. One idea would be to add an env var to turn off the testcontainers tests and set it in appveyor but not in GH actions. What do you think?

Reply:
so the issue here came from trying to start the reaper containers (the containers that run and ensure that the containers we start get killed at the end of the tests), but you can optionally skip the reaper containers.

I think that since, according to appveyor's documentation, they don't preserve any VMs between builds, we can safely skip the reaper in appveyor tests.  So I think we should add that env variable like `TESTCONTAINERS_SKIP_REAPER`, if that is set to `1` then we skip the reaper builds and we should be okay (_I think_)

Reply:
I am testing this theory out now...I accidentally got a build stuck debugging...so I am going to wait for that to timeout and fail to keep on testing

Reply:
hey @arschles how do you feel about putting integration tests into their own package? the using a build tag to ignore them by default, then you can pass in the tag to enable them?  that should be able to be done just from the code and the config files and wouldn't require a change to the environment

Reply:
Fine with me RE the different package. I'd rather have them enabled by default and pass a tag to disable them. Are you cool with that?

Reply:
yeah that makes sense to me, just need a way to cleanly include/not include them

I can work on this

Reply:
That's great! We had an idea in #1500 to not turn off the tests completely, but just to turn off the testcontainers logic based on a tag. I laid it out [here](https://github.com/gomods/athens/issues/1500#issuecomment-597381000)

What do you think about that?

Reply:
just left a comment